pandas/core/base.py

```diff
@@ -965,6 +977,12 @@ def is_unique(self) -> bool:
         Returns
         -------
         bool
+
+        Examples
+        --------
+        >>> s = pd.Series([1, 2, 3, 1])
+        >>> s.is_unique
+        False
```

Comment:
this one also returns `bool` ;) rest looks good though!

Reply:
I should have seen it :) .. thanks